bin/git-bulk

```diff
@@ -141,7 +142,6 @@ function executBulkOp () {
       eval cd "\"$gitrepodir\"" # into git repo location
       local curdir=$(pwd)
       local leadingpath=${curdir#${actual}}
-      [ "${quiet?}" != "true" ] && echo 1>&2 "Current repository: ${leadingpath%/*}/${bldred}${curdir##*/}${reset}"
```

Comment:
Why remove this prompt?

Reply:
It is included now in the the line where it prints: "Executing git something". Now it it

"Executing git something in repository yourrepo"

Reply:
I see